packages/react-native-windows-init/src/Cli.ts

```diff
@@ -274,36 +277,53 @@ async function getLatestMatchingRNWVersion(
   }
 }
 
-function installReactNativeWindows(version: string, useDevMode: boolean) {
-  console.log(
-    `Installing ${chalk.green('react-native-windows')}@${chalk.cyan(
-      version,
-    )}...`,
-  );
-
+function installReactNativeWindows(
+  version: string | undefined,
+  useDevMode: boolean,
+) {
   const cwd = process.cwd();
   const execOptions = argv.verbose ? {stdio: 'inherit' as 'inherit'} : {};
 
   if (useDevMode) {
-    execSync(
-      isProjectUsingYarn(cwd)
-        ? 'yarn link react-native-windows'
-        : 'npm link react-native-windows',
-      execOptions,
+    const packageCmd = isProjectUsingYarn(cwd) ? 'yarn' : 'npm';
+    execSync(`${packageCmd} link react-native-windows`, execOptions);
+    const rnwPkgJsonPath = path.resolve(
+      './node_modules/react-native-windows/package.json',
     );
```

Comment:
This path won't always be correct in cases of hoisted modules. Consider instead:

```suggestion
    const rnwPkgJsonPath = require.resolve('react-native-windows/package.json',
      {paths: [cwd}
    );
```

Reply:
Thanks, I tried just regular import and `require` directly but that did not find the file properly...